roles/rollup/templates/restore-from-snapshot.sh.j2

```diff
@@ -0,0 +1,308 @@
+#!/bin/bash
+#
+# Restore Rollup Data from EBS Snapshot or Volume
+# This script restores rollup data from an EBS snapshot or pre-created volume to local NVMe storage.
+# Usage: ./restore-from-snapshot.sh <snapshot-id|volume-id> [--force]
+#
+set -e
+
+ROLLUP_DATA_DIR="{{ rollup_storage_dir }}"
+TEMP_MOUNT="/mnt/restore-temp"
+TEMP_DEVICE_NAME="/dev/sdr"  # Unlikely to conflict with existing devices
+LOGFILE="/var/log/rollup-restore.log"
```

Comment:
Nit: but maybe better to use {{ logs_dir }} here, so all logs are in the same place?